src/components/m-dropdown/settings/dropdown.scss

```diff
@@ -6,7 +6,7 @@ $dropdown__frame-color: $color-prim-gray-mine-shaft !default;
 $dropdown__frame-color--hover: $color-axa-blue !default;
 $dropdown__frame-y-padding: 10.5px !default;
 $dropdown__frame-y-padding-sm: 5.5px !default;
-$dropdown__frame-x-padding: 20px !default;
+$dropdown__frame-x-padding: 12px !default;
```

Comment:
@jackblackCH 
@LucaMele 
Is this reviewed and approved in general by Design/UX.
Because we have a 10-based grid

Reply:
I suggest we discuss topics like that with designers. I highly agree on pixel perfect solutions. Nevertheless, pixels in CSS are never accurate as they are always a combo of different properties and side effects.

Reply:
Design changes are discussed. @jojackpot agreed.

Reply:
As long as it is approved by Design/UX I'm okay with it

Reply:
perfect, so ok for me for this change